packages/style/src/index.js

```diff
@@ -2,16 +2,46 @@ import React from 'react'
 import niceStyles from 'motion-nice-styles'
 import { StyleSheet, css } from 'aphrodite'
 
-export default function Style(ComposedComponent) {
-  class StyledComponent extends ComposedComponent {
+let opts = {
+  theme: true,
```

Comment:
I should not ideally you do this one. Before I had it once-per component (why I asked you about detecting react components because I didn't want to force people to write `@style()` every time.